app/views/markets/show.html.erb

```diff
@@ -8,10 +8,10 @@
   FNAP: <%= @market.market.fnap %> <br>
   SNAP: <%= @market.market.snap %> <br>
   Accepted Payments: <%= @market.market.payment %> <br>
-  latitude: <% @market.market.latitude %>
-  longitude: <% @market.market.longitude %>
+  latitude: <%= @market.market.latitude %> <br>
```

Comment:
@market.market doesn't tell me a lot about what's happening here. I had to go hunt down the train of code in all of the files to understand what was happening here. In the spirit of self-commenting code, can we call the market method something more descriptive? Or call @market @market_facade?